test/tornado_tests/test_motor_cursor.py

```diff
@@ -279,9 +279,11 @@ async def test_cursor_explicit_close(self):
 
         self.assertTrue((await fetch_next))
 
-        await cursor.close()
-        request = await self.run_thread(server.receives, "killCursors", "coll")
-        request.ok()
+        async def mock_kill_cursors():
+            request = await self.run_thread(
+                server.receives, "killCursors", "coll")
+            request.ok()
+        await gen.multi([cursor.close(), mock_kill_cursors()])
```

Comment:
This was required because of the MotorCursor.close changes. Close() now waits for killCursors to complete so we need to account for that in this test.